tests/test_holders.py

```diff
@@ -139,7 +139,7 @@ def test07_uniqueptr_passthrough(clean):
     t.reset()
 
     with pytest.warns(RuntimeWarning, match=r'nanobind::detail::nb_relinquish_ownership()'):
-        with pytest.raises(TypeError) as excinfo:
+        with pytest.raises(TypeError):
             assert t.passthrough_unique(t.Example(1)).value == 1
```

Comment:
```suggestion
with pytest.warns(RuntimeWarning, match=r'nanobind::detail::nb_relinquish_ownership()'), pytest.raises(TypeError):
        assert t.passthrough_unique(t.Example(1)).value == 1
```

Reply:
You have to make the suggestion on the new line, not the deleted line, for it to be automatically apply-able.